native-sql-engine/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -1,729 +0,0 @@
-/*
```

Comment:
Pending.
This file is ported from spark 3.1 source code with few places adding `existingStage.schema == e.schema`.